WoWsShipBuilder.Common/Features/ShipSelection/ShipSelector.razor

```diff
@@ -345,7 +364,12 @@
 
         if (result && !string.IsNullOrWhiteSpace(textSearch))
         {
-            result = CultureInfo.CurrentCulture.CompareInfo.IndexOf(valuePair.Key, textSearch, CompareOptions.IgnoreNonSpace | CompareOptions.IgnoreCase) != -1;
+            result = CultureInfo.CurrentCulture.CompareInfo.IndexOf(shipName, textSearch, CompareOptions.IgnoreNonSpace | CompareOptions.IgnoreCase) != -1;
```

Comment:
This should not use CultureInfo.CurrentCulture but instead use the selected user culture from app settings.